### ISSUE TITLE
Add, enhance, correct landmarks (warnings; moderate impact)

### ISSUE THREAD

**Issue by jvine** [sul-dlss/exhibits#1619]
Goal is to make all sections of the page available to a SR-user who uses landmarks, and disambiguate multiple landmarks of the same type (e.g. nav).

Working from top down in the HTML.

### Skip links
- [x] change div to nav, add role and aria-label
`<div id="skip-link">`
`<nav id="skip-link" role="navigation" aria-label="Skip links">`

### Topbar
- [x] add aria-label (same as Arclight)
`<nav class="... topbar" role="navigation">`
`<nav class="... topbar" role="navigation" aria-label="Utilities">`

### Masthead
- [x] change div to header, add role
`<div class="masthead image-masthead ">`
`<header class="masthead image-masthead " role="banner">`

### Exhibit navigation
- [x] add aria-label
`<div id="exhibit-navbar" ... role="navigation">`
`<div id="exhibit-navbar" ... role="navigation" aria-label="Exhibit navigation">`

### Search form
- [x] add aria-label
`<form class="search-query-form" role="search" ...>`
`<form class="search-query-form" role="search" aria-label="Search" ...>`

### Breadcrumb
- [x] change div to nav, add role and aria-label (same as Arclight)
`<div class="container breadcrumbs-container">`
`<nav class="container breadcrumbs-container" role="navigation" aria-label="Breadcrumb">`

### Main 
- is fine!

### Search results section
- [x] add aria-label (same as Arclight)
`<section id="content" class="col-lg-9 order-last">`
`<section id="content" class="col-lg-9 order-last" aria-label="Search results">`

### Search results nav
- [x] add role and aria-label
`<div id="sortAndPerPage" ...>`
`<div id="sortAndPerPage" ...  role="navigation" aria-label="Results navigation">`

### Pagination
- [x] correct role, give more context to aria-label
`<nav class="pagination" role="region" aria-label="pagination links">`
`<nav class="pagination" role="navigation" aria-label="Results pagination">`

### Sidebar
- [x] add aria-label (same as Arclight)
`<aside id="sidebar" class="page-sidebar col-lg-3 order-first">`
`<aside id="sidebar" class="page-sidebar col-lg-3 order-first" aria-label="Limit your search">`

### Footer
Technically, there should be only one footer landmark. We have two:
1. `<footer role="contentinfo">` (SUL footer)
2. `<div id="global-footer"  role="contentinfo">` (global SU footer)

**Option A**:  Nest both divs within one footer - but this might break the already brittle footer layout on mobile. 
```
<footer role="contentinfo">
  <div id="sul-footer"> ... </div>
  <div id="global-footer"> ... </div>
</footer>
```

**Option B**:  Live with two, but add aria-labels to differentiate.
1. `<footer role="contentinfo" aria-label="Stanford Libraries footer">` 
2. `<div id="global-footer"  role="contentinfo" aria-label="Stanford University footer">`

**Comment by mejackreed** [comment]
@jvine are we good to close this now?

**Comment by jvine** [comment]
All good.
